actionview/karma.conf.js

```diff
@@ -0,0 +1,64 @@
+const config = {
+  browsers: ["ChromeHeadless"],
+  frameworks: ["qunit"],
+  files: [
+    "test/ujs/compiled/test.js",
+  ],
+
+  client: {
+    clearContext: false,
+    qunit: {
+      showUI: true
+    }
+  },
+
+  singleRun: true,
+  autoWatch: false,
+
+  captureTimeout: 180000,
+  browserDisconnectTimeout: 180000,
+  browserDisconnectTolerance: 3,
+  browserNoActivityTimeout: 300000,
+  proxies: {
+    '/echo': 'http://localhost:4567/echo',
+    '/error': 'http://localhost:4567/error'
+  }
+}
+
+if (process.env.CI) {
+  config.customLaunchers = {
+    sl_chrome: sauce("chrome", "latest", "Windows 10")
```

Comment:
What does "Windows 10" mean in this context?

Reply:
It's referring to the `sauce(browserName, version, platform)` function on line 42.